dev/com.ibm.websphere.appserver.features/visibility/public/servlet-6.0/com.ibm.websphere.appserver.servlet-6.0.feature

```diff
@@ -61,7 +61,7 @@ Subsystem-Category: JakartaEE10Application
  com.ibm.ws.http.plugin.merge, \
  com.ibm.ws.webserver.plugin.runtime.jakarta, \
  com.ibm.ws.webserver.plugin.runtime.interfaces,\
- io.openliberty.servlet.6.0.internal.factories.jakarta
```

Comment:
io.openliberty.servlet.6.0.internal.factories is not a `pure` jakarta as one class still has an javax.servlet import

Reply:
I can change the import to jakarta.servlet after servlet API PR is merged

Reply:
It is already changed in this PR.